src/emulator/functionsEmulator.ts

```diff
@@ -271,7 +271,7 @@ export class FunctionsEmulator implements EmulatorInstance {
 
     // The URL for the function that the other emulators (Firestore, etc) use.
     // TODO(abehaskins): Make the other emulators use the route below and remove this.
-    const backgroundFunctionRoute = `/functions/projects/:project_id/triggers/:trigger_name`;
+    const backgroundFunctionRoute = `/functions/projects/:project_id/triggers/:trigger_name(*)`;
```

Comment:
why did we need this change"

Reply:
trigger_name is of format <function name>-<fully qualified channel name with forward slashes> so it doesn't parse the path correctly unless we specify that the rest of the path should be used for trigger_name.

Reply:
Err, tags got stripped. Meant to say of format "function name" - "fully qualified channel path with forward slashes".